app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,8 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
 
-    <string name="noGiftCards">Click the "+" (plus) button to add a card first.\n\nCatima carries your cards on the device, so they are always within reach.</string>
-    <string name="noMatchingGiftCards">Didn\'t find anything. Try changing your search.</string>
+    <string name="noGiftCards">Click the "+" (plus) button to add a card first.</string>
```

Comment:
```suggestion
    <string name="noGiftCards">Click the ⊕ plus button to add a card, or import some from the ⋮menu first.</string>
```

Reply:
Should probably need a space after the 3 dots but otherwise sounds good yeah